wordcamp.test/provision/vvv-init.sh

```diff
@@ -74,19 +58,27 @@ else
 
 	git -C $SITE_DIR pull origin master
 
-	for i in "${SVN_PLUGINS[@]}"
-	do :
+	wme_noroot wp core   update                     --path=$SITE_DIR/wordpress
+	wme_noroot wp plugin update ${WPCLI_PLUGINS[@]} --path=$SITE_DIR/wordpress
+	wme_noroot wp theme  update ${WPCLI_THEMES[@]}  --path=$SITE_DIR/wordpress
+fi
+
+
+for i in "${SVN_PLUGINS[@]}"
+do :
+	if [ ! -L "$SITE_DIR/wp-content/plugins/$i" ]; then
+		svn co https://plugins.svn.wordpress.org/$i/trunk "$SITE_DIR/wp-content/plugins/$i"
+	else
 		(
 			cd "${SITE_DIR}/wp-content/plugins/$i" &&
 			svn cleanup &&
 			svn up
 		)
-	done
+	fi
+done
 
+if [ ! -L "$SITE_DIR/wp-content/plugins/camptix" ]; then
+	git clone https://github.com/Automattic/camptix.git $SITE_DIR/wp-content/plugins/camptix
+else
```

Comment:
What's the reason to move these 3 sections around?

Reply:
Resilience. If for whatever reason the wordcamp.test provisioner fails half way through, then the site is in limbo. It'll never svn checkout all the remaining items and install the remaining plugins because the folder already exists. Instead it'll try to update them and fail.

So, moving svn checkouts/updates and camptix clone/pull into their own separate items ensures that if it fails you can just reprovision without having to nuke the entire WP Meta environment

Reply:
At, that's a great idea :)